### PR TITLE
feat: Add scroll to all records in dashboard

### PR DESCRIPTION
**Pull request by cripacrip**

### CODE REVIEW

File: src/pages/records/list.vue [hunk 1]
```diff
@@ -46,6 +46,10 @@ export default defineComponent({
 </script>
 
 <style lang="scss" scoped>
+.transactions-list {
+  overflow-y: scroll;
+  max-height: calc(100vh - 160px);
+}
```
Comment (letehaha) [inline src/pages/records/list.vue:52]: Why do we need `scroll` and max-height here? We already have a scroll in the `.records` selector.

File: src/pages/records/records.vue [hunk 2]
```diff
@@ -53,14 +53,18 @@ export default defineComponent({
...
 .records__list {
   background-color: var(--app-surface-color);
   border-radius: 8px;
   padding: 24px;
 
   max-width: 560px;
+  max-height: 100%;
   margin: 0 auto;
+
+  overflow-y: hidden;
```
Comment (letehaha) [inline src/pages/records/records.vue]: Is that really needed?

**Review by letehaha — CHANGES_REQUESTED** [review]

File: src/styles/variables.scss [hunk 3]
```diff
@@ -59,7 +59,7 @@ body {
   // sizes
   --container-width: 1200px;
   --container-padding: 32px;
-  --header-height: 61px;
+  --header-height: 77px;
```
Comment (letehaha) [inline src/styles/variables.scss:62]: This variable is only used with your records list, but not with the header itself. So currently header's height is differ from that value

**Review by letehaha — CHANGES_REQUESTED** [review]

File: src/styles/variables.scss (hunk 3 above)
Comment (cripacrip) [reply src/styles/variables.scss:62]: And what i need to do with that variable, like, create new for this record list?

**Review by letehaha — APPROVED** [review]

**Pull request merged**